src/controller/orderController.js

```diff
@@ -1,15 +1,19 @@
+const jwt = require("jsonwebtoken");
 const cartModel = require("../models/cart");
 const productModel = require("../models/product");
 const orderModel = require("../models/order");
+const appConfig = require("../config/appConfig");
 
 // Create a Order
 exports.checkOut = async (req, res) => {
+  const { userId } = jwt.decode(req.cookies.jwtToken, appConfig.jwtKey);
```

Comment:
why we add logic here

Reply:
it takes token from cookie and decode it for take userid from token